boot/seal.go

```diff
@@ -406,35 +406,35 @@ func resealKeyToModeenvSecboot(rootdir string, model *asserts.Model, modeenv *Mo
 		return fmt.Errorf("cannot compose run mode boot chains: %v", err)
 	}
 
```

Comment:
we need a TODO in  recoveryBootChainsForSystems to fix how it's using model, we probably need to pass more than one model in the testing case